test/puppetlabs/puppetdb/http/catalogs_test.clj

```diff
@@ -103,7 +106,7 @@
             expected (get paging-options p)]
         (is (= (map :certname expected) (map :certname response-body)))))))
 
-  (testing "/v4 endpoint is still responsive to old-style node queries"
-    (let [{:keys [body]} (get-response "/v4/catalogs" "myhost.localdomain")
+  (testing "endpoint is still responsive to old-style node queries"
+    (let [{:keys [body]} (get-response endpoint "myhost.localdomain")
```

Comment:
this functionality (curling /v4/catalogs/hostname) seems completely broken on postgres. Not sure how this test passes.